### PR TITLE
Add appbar layout

### PR DESCRIPTION
**Pull request by Darius3001**

### CODE REVIEW

File: demo/src/main/java/de/check24/compose/demo/features/appbarlayout/ComposableAppBarLayoutActivity.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+private fun AppBarExample() {
+    TopAppBar {
+        IconButton(onClick = { /* your code */ }) {
+            Icon(
+                imageVector = Icons.Rounded.Menu,
+                contentDescription = "Menu",
+                modifier = Modifier.padding(10.dp)
+            )
+        }
+        Text(
```
Comment (Justus-Saringer) [inline demo/src/main/java/de/check24/compose/demo/features/appbarlayout/ComposableAppBarLayoutActivity.kt:42]: Why don't you use a scaffold? Then you can use the title and navigationitem from the TopAppBar?

File: demo/src/main/res/layout/app_bar_layout.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/appbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (Justus-Saringer) [inline demo/src/main/res/layout/app_bar_layout.xml:12]: Why do you use a ConstraintLayout in the AppBarLayout?

**Review by ColdTea-Projects — APPROVED** [review]

File: code/container/AppBarLayout.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+&lt;com.google.android.material.appbar.AppBarLayout
+    android:id="@+id/appbar"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"&gt;
+
+    &lt;androidx.constraintlayout.widget.ConstraintLayout
```
Comment (Tautellini) [inline code/container/AppBarLayout.xml:6]: See comment from @Justus-Saringer

File: demo/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -15,4 +15,5 @@
     <string name="shadow">shadow</string>
     <string name="modifier_background">red background</string>
     <string name="check24logo">Check24 Logo</string>
```
Comment (Tautellini) [inline demo/src/main/res/values/strings.xml:17]: Nothing too important for me, and not your change, but you should stay consistent for your namings.

File: demo/src/main/res/layout/app_bar_layout.xml (hunk 2 above)
Comment (Darius3001) [reply demo/src/main/res/layout/app_bar_layout.xml:12]: Because its vertical. Was my approach to make it horizontal

**Pull request merged**